nise/yaml_generators/oci/oci_yaml_constants.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""OCI Yaml File Constants"""
+from dataclasses import dataclass
+from dataclasses import field
+
+from faker import Faker
+
+
+fake = Faker()
+OCI_TAGS = {
+    "storage": ["tags/new-tags.tarnished-tags", "tags/orcl-cloud.free-tier-retained"],
+    "compute": ["tags/free-form-tag", "tags/orcl-cloud.free-tier-retained"],
+    "database": ["tags/free-form-tag"],
+    "network": ["tags/free-form-tag"],
+}
+
+
+@dataclass(frozen=True)
+class OCIYamlConstants:
+    """OCI yaml constant columns"""
+
+    tenant_id: str = field(default="ocid1.tenancy.oc1..EfjkUPxyZSYLvd")
+    compartment_name: str = field(default=fake.name().replace(" ", "").lower())
+    subscription_id: int = field(default=fake.random_number(fix_len=True, digits=8))
+    tag_keys: dict = field(default=lambda key: OCI_TAGS.get(key))
```

Comment:
This should be a method. This works by assigning a callable to an attribute, but that's very atypical.
```suggestion

   def get_tag_keys(self, key: str) -> t.List[str]:
       return OCI_TAGS.get(key)
```

Reply:
If `OCI_TAGS` is changed to a dataclass:
```python
def get_tag_keys(self, key: str) -> t.List[str]:
    return getattr(OCITags, key)